src/workerd/api/node/tests/buffer-nodejs-test.js

```diff
@@ -5792,3 +5793,104 @@ export const transcodeTest = {
     }
   }
 };
+
+// Tests are taken from Node.js
+// https://github.com/nodejs/node/blob/main/test/parallel/test-file.js
+export const fileTest = {
+  test() {
+    throws(() => new File(), TypeError);
+    throws(() => new File([]), TypeError);
+    throws(() => File.prototype.name, TypeError);
+    throws(() => File.prototype.lastModified, TypeError);
+
+    {
```

Comment:
Not sure if possible with the test framework here, but it would be nice to different tests, each with a description - there are already blocks scoping the test cases, it would be nice to add description and different fn (i.e. à la Jest `describe` / `test`)

Reply:
They can be split into individual `export const fooTest = { test() { ... } }` blocks